### ISSUE TITLE
Import contacts and companies directly from LinkedIn

### ISSUE THREAD

**Issue by florrush** [twentyhq/twenty#796]
Currently testing various CRMs (see folk.app as an example) where you can either import individual contacts or lists directly from LinkedIn and add them to a group within the CRM. This usually pulls any available data, including the url link and email.  Ideally would be able to carry over list of Companies too.

**Comment by FelixMalfait** [comment]
Yes this is definitely something we need to build! But it's unlikely we would prioritize it in the next 3months since we'll remain focused on foundational stuff. Would you see this as a Chrome extension?
